contrib/kind.sh

```diff
@@ -519,12 +522,36 @@ set_default_params() {
   JOIN_SUBNET_IPV4=${JOIN_SUBNET_IPV4:-100.64.0.0/16}
   JOIN_SUBNET_IPV6=${JOIN_SUBNET_IPV6:-fd98::/64}
   KIND_NUM_MASTER=1
+  OVN_INTERCONNECT_ENABLE=${OVN_INTERCONNECT_ENABLE:-false}
+  KIND_NUM_ZONE_NODES=${KIND_NUM_ZONE_NODES:-1}
```

Comment:
NIT: Can we rename it `KIND_NUM_NODES_PER_ZONE` to make it extra clear what it is for?

Reply:
+1 to PER_ZONE, I had the same comment I wanted to give but since this PR is after 3169 was patient :D